app/scripts/proactive/templates/catalog-set-templates-bucket.html

```diff
@@ -5,7 +5,7 @@
                 <fieldset>
                     <legend style="font-weight: bold;">Buckets</legend>
                     <div style="height: 270px; overflow: auto;">
-                        <table id="catalog-set-templates-<%- this.order %>-bucket-table" style="width: 100%; cursor: pointer;"></table>
+                        <table id="catalog-set-templates-secondary-bucket-table" style="width: 100%; cursor: pointer;"></table>
```

Comment:
Are you sure this won't result in several elements with the same ID?

Reply:
no, because this is a table, and there is only one table in the Add bucket to palette modal
Besides, `<%- this.order %>` used to be either `main` or `secondary`.
There is no `main buckets in the palette anymore` you can freely do whatever you want.
the only menu you can't remove is `Tasks`